controllers/modelmesh/runtime.go

```diff
@@ -248,43 +249,44 @@ func (m *Deployment) addRuntimeToDeployment(deployment *appsv1.Deployment) error
 			{}, {}, {}, {}, // allocate larger array to avoid reallocation
 		}[:8]
 
-		if mlc, ok := rt.Annotations["maxLoadingConcurrency"]; ok {
-			builtInAdapterContainer.Env = append(builtInAdapterContainer.Env, corev1.EnvVar{
-				Name:  "LOADING_CONCURRENCY",
-				Value: mlc,
-			})
-		}
+		if rta != nil {
```

Comment:
I think this `if` could be removed now, could revert to be closer to the original code (just `owner` instead of `rt`)